Nos/Views/RepliesView.swift

```diff
@@ -90,11 +90,13 @@ struct RepliesView: View {
             let subID = await relayService.openSubscription(with: filter)
             subscriptionIDs.append(subID)
             
-            // download reports for this user
+            // download reports for this user and the replies' authors
             guard let authorKey = note.author?.hexadecimalPublicKey else {
                 return
             }
-            let reportFilter = Filter(kinds: [.report], pTags: [authorKey])
+            let pTags = [authorKey] + replies.compactMap { $0.author?.hexadecimalPublicKey }
```

Comment:
Not sure how relays will behave if there are duplicate author keys in this array. Can we do a .removeDuplicates on it or something?